### PR TITLE
Mocha local & pass obt verify

### PR DESCRIPTION
**Pull request by kaelig**
- ~~Don't require mocha to be installed globally~~
- Pass Linting

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,4 +6,4 @@ install:
 test:
 	origami-build-tools verify --verbose
 	origami-build-tools build --sass=tests/test.scss --buildCss=test.css --env=production
-	mocha tests/sass-setup.spec.js
+	./node_modules/mocha/bin/mocha tests/sass-setup.spec.js
```
Comment (matthew-andrews) [inline Makefile]: plz revert.

Comment (matthew-andrews) [inline Makefile]: see notes here:- 
http://financial-times.github.io/next/docs/developer-guide/troubleshooting/

> Can’t find nodemon, mocha, etc. Your path might be missing node_modules/.bin. Edit your .bashrc or .bash_profile file and ensure your path has the following paths added to it:-
export PATH="node_modules/.bin:$PATH" Then run source ~/.bash_profile to add it to your current terminal session

(we don't require mocha to be install globally)

Comment (kaelig) [inline Makefile]: Sorry about that!

Comment (kaelig) [inline Makefile]: Reverted in e936420, thank you for pointing it.

**Pull request merged**